ai-platform-unified/notebooks/unofficial/sdk/metric-parameter-tracking-for-custom-jobs.ipynb

```diff
@@ -0,0 +1,1006 @@
+{
```

Comment:
<p>You should use the same cell that the template provides and not hardcode to "python-docs-samples-tests". This is the testing CI's project. Perhaps you accidentally checked in this change when you ran the test script locally.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/310/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>Actually this cell was copied from the template, which also has "python-docs-samples-tests" hard coded. Shall we update this to something else instead?</p><p>https://github.com/GoogleCloudPlatform/ai-platform-samples/blob/master/ai-platform-unified/notebooks/notebook_template.ipynb</p>

Reply:
<p>Updated this to "[your-project-id]" per https://github.com/GoogleCloudPlatform/ai-platform-samples/blob/master/ai-platform-unified/notebooks/official/custom/sdk-custom-image-classification-online.ipynb</p>